client/css/style.css

```diff
@@ -573,6 +573,7 @@ button {
 }
 #chat .wrap,
 #chat .text a {
+	cursor: pointer;
```

Comment:
Sorry, I didn't see the `#chat .wrap` on the first line. However, I'm confused, all links are wrapped into `a` links (what you called normal HTML links). Do you have an example where we need this?